config/conference.ts

```diff
@@ -71,6 +71,8 @@ if (!hideDate) {
   })
 }
 
+const venue = null // Venue
```

Comment:
This will likely break the Venue Info page (/venue). Is there a specific reason this is being reset to null here, or are we taking down that page at the same time as well? You may also want to adjust related content on /conference-day as well, which includes some venue information.

Reply:
Hrrm, strict mode isn't enabled. Something for the backlog. Fixed. Conference day looks ok